### ISSUE TITLE
🐛 [Bug] Lakeshire Guard incorrect progression level

### ISSUE THREAD

**Issue by Daribon** [vmangos/core#1735]
## 🐛 Bug report

Lakeshire Guard are currently level 55 no matter what server patch you got set.
They should be level 35 in the early patches.

### Expected behavior

On Allakhazam Lakeshire Guard is level 35 until December 2005 when it changed to level 55:
https://web.archive.org/web/20050524000758/http://wow.allakhazam.com/dyn/mobs/zone33.html

Goblin Workshop lists Lakeshire Guard as level 35 in March 2005:
https://web.archive.org/web/20050307102945/http://www.goblinworkshop.com/monsters.html?letter=L
In May 2006, Goblin Workshop has Lakeshire Guard as level 55.

There is a big gap between March and December, however considering the other guards that got changed in patch 1.7.0 (September 2005) Lakeshire Guard most likely got changed during same patch.

Listed as level 35 at Judgehype:
https://web.archive.org/web/20060505122916/http://worldofwarcraft.judgehype.com/index.php?page=Cmonstre_zone&cz=1

I have no in game image of them being level 35 yet, but I'm working on going through 5k+ 2005 screenshots.

**TL;DR:
Lakeshire Guard should most likely be level 35 until patch 1.7.0**

### Steps to reproduce

1. Set server to low patch
2. Login on a Horde character
3. .tele Lakeshire
4. Target Lakeshire Guard
5. Profit

### Version & Environment

Client Version: 1.12.1.5875

Commit Hash:  https://github.com/vmangos/core/commit/4d7e82f4c14cb4360f21122ede6d3d1c0b7613cb

OS Client: Microsoft Windows 11 Pro N 10.0.22621.900
OS Server: Microsoft Windows 11 Pro N 10.0.22621.900

### Crashlog

- None
